snowflake/pkg/infra/udfs.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2022 Antrea Authors.
```

Comment:
I am not sure this needs to be under `pkg/infra` at all. How about `pkg/udfs`?

Reply:
Sounds good, moved it there.